real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/RobotLogger.java

```diff
@@ -19,17 +22,24 @@
  * @author Trevor Decker
  */
 public final class RobotLogger {
-	public  static Logger message;
-	public static SensorLogger sensor;
+	private static Logger message;
+	private static SensorLogger sensor;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1870702230/issues/source?bid=2938665&fileBranchId=3087231#l26)